### PR TITLE
ML-004/add country

### PR DESCRIPTION
**Pull request by twill11**

### CODE REVIEW

File: spec/factories/countries.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  factory :country do
+    title { 'England' }
+
+    trait :invalid_short_title do
+      title { 'Ya' }
```
Comment (WhatWasThatt) [inline spec/factories/countries.rb:6]: use Faker::Lorem.characters(number: 2)

File: app/models/country.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class Country < ApplicationRecord
+  validates :title, presence: true, length: { minimum: 3, maximum: 25 }
```
Comment (WhatWasThatt) [inline app/models/country.rb:2]: change maximum to 100
image

**Review by KarinaKlevzhits — COMMENTED** [review]
I think it's better to use Faker in factory.Good Job!

**Review by KarinaKlevzhits — APPROVED** [review]

**Review by WhatWasThatt — APPROVED** [review]
Thanks for your changes. Good job!

**Review by ShrewHools — APPROVED** [review]

**Comment by ShrewHools** [comment]
gj

**Pull request merged**